.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,36 @@
+name: "CodeQL Code Scan"
```

```suggestion
name: "CodeQL"
```

That should show the code scan result under `Security` tab?

---

Yep it if is enabled in the security tab, the action will post to the alerts

---

Do we need to click on that in the `Security` tab or do we need to rename it to `CodeQL` ?